### ISSUE TITLE
ImportError: cannot import name 'is_xformers_available' from 'diffusers.utils.import_utils'

### ISSUE THREAD

**Issue by pandasonic-collective** [d8ahazard/sd_dreambooth_extension#272]
## Kindly read this ENTIRE form and fill it out with the requested information. It may seem tedious, but it's even more tedious asking basic questions to ensure I'm not trying to troubleshoot a bug I've already fixed. Help me help you, answer all the questions and follow the steps below before posting.

**Please find the following lines in the console and paste them below. _If you do not provide this information, your issue will be automatically closed._**

`
Python 3.10.8 (tags/v3.10.8:aaaf517, Oct 11 2022, 16:50:30) [MSC v.1933 64 bit (AMD64)]
Commit hash: 828438b4a190759807f9054932cae3a8b880ddf1
Installing requirements for Web UI

Error running install.py for extension extensions\sd_dreambooth_extension.
ImportError: cannot import name 'is_xformers_available' from 'diffusers.utils.import_utils'
`

**Have you read the [Readme](https://github.com/d8ahazard/sd_dreambooth_extension#readme)?**
yes
**Have you completely restarted the stable-diffusion-webUI, not just reloaded the UI?**
yes
**Have you updated Dreambooth to the latest revision?**
yes
**Have you updated the Stable-Diffusion-WebUI to the latest version?**
yes
**No, really. Please save us both some trouble and update the SD-WebUI and Extension and restart before posting this. Reply 'OK' Below to acknowledge that you did this.**
OK...

**Describe the bug**
See Above, that's as much as I have.
I just pulled the latest about 5 mins ago

(A clear and concise description of what the bug is)

**Provide logs**
Error loading script: api.py
Traceback (most recent call last):
  File "S:\Stable Diffusion\stable-diffusion-webui\modules\scripts.py", line 184, in load_scripts
    module = script_loading.load_module(scriptfile.path)
  File "S:\Stable Diffusion\stable-diffusion-webui\modules\script_loading.py", line 13, in load_module
    exec(compiled, module.__dict__)
  File "S:\Stable Diffusion\stable-diffusion-webui\extensions\sd_dreambooth_extension\scripts\api.py", line 9, in <module>
    from extensions.sd_dreambooth_extension.dreambooth import conversion, dreambooth
  File "S:\Stable Diffusion\stable-diffusion-webui\extensions\sd_dreambooth_extension\dreambooth\conversion.py", line 31, in <module>
    from extensions.sd_dreambooth_extension.dreambooth import xattention
  File "S:\Stable Diffusion\stable-diffusion-webui\extensions\sd_dreambooth_extension\dreambooth\xattention.py", line 9, in <module>
    from diffusers.models.attention import xformers
ImportError: cannot import name 'xformers' from 'diffusers.models.attention' 

Error loading script: main.py
Traceback (most recent call last):
  File "S:\Stable Diffusion\stable-diffusion-webui\modules\scripts.py", line 184, in load_scripts
    module = script_loading.load_module(scriptfile.path)
  File "S:\Stable Diffusion\stable-diffusion-webui\modules\script_loading.py", line 13, in load_module
    exec(compiled, module.__dict__)
  File "S:\Stable Diffusion\stable-diffusion-webui\extensions\sd_dreambooth_extension\scripts\main.py", line 3, in <module>
    from extensions.sd_dreambooth_extension.dreambooth import conversion, dreambooth
  File "S:\Stable Diffusion\stable-diffusion-webui\extensions\sd_dreambooth_extension\dreambooth\conversion.py", line 31, in <module>
    from extensions.sd_dreambooth_extension.dreambooth import xattention
  File "S:\Stable Diffusion\stable-diffusion-webui\extensions\sd_dreambooth_extension\dreambooth\xattention.py", line 9, in <module>
    from diffusers.models.attention import xformers
ImportError: cannot import name 'xformers' from 'diffusers.models.attention'

If a crash has occurred, please provide the *entire* stack trace from the log, including the last few log messages *before* the crash occurred.

**Environment**

What OS? Win11

If Windows - WSL or native? Native

What GPU are you using? 3090ti

**Screenshots/Config**
If the issue is specific to an error while training, please provide a screenshot of training parameters or the db_config.json file from /models/dreambooth/MODELNAME/db_config.json

**Comment by CobbleGen** [comment]
Having this exact same issue on win 10. Have tried different versions of python but to no avail. Also tried adding `--xformers` as an argument, but made no difference.

My error starts with this above the exact same error messages you are having:

```
Error running install.py for extension extensions\sd_dreambooth_extension.
Command: "F:\Installations\SD-webui\stable-diffusion-webui\venv\Scripts\python.exe" "extensions\sd_dreambooth_extension\install.py"
Error code: 1
stdout: <empty>
stderr: Traceback (most recent call last):
  File "F:\Installations\SD-webui\stable-diffusion-webui\extensions\sd_dreambooth_extension\install.py", line 14, in <module>
    from diffusers.utils.import_utils import is_xformers_available
ImportError: cannot import name 'is_xformers_available' from 'diffusers.utils.import_utils' (F:\Installations\SD-webui\stable-diffusion-webui\venv\lib\site-packages\diffusers\utils\import_utils.py)
```

**Comment by FloydianSound** [comment]
same error here too, Windows 10, Cuda 11.8. python 3.10.8
![image](https://user-images.githubusercontent.com/97499891/202932959-4940b4e7-f50c-4b93-882e-70c61bf45606.png)

**Comment by d8ahazard** [comment]
Yep. I didn't think this one out entirely.

Should be fixed with 58bb667

**Comment by trihardseven** [comment]
@d8ahazard  This is still happening with latest version:
venv "H:\stable-diffusion-webui\venv\Scripts\Python.exe"
Python 3.10.6 (tags/v3.10.6:9c7b4bd, Aug  1 2022, 21:53:49) [MSC v.1932 64 bit (AMD64)]
Commit hash: 828438b4a190759807f9054932cae3a8b880ddf1
Installing requirements for Web UI
Error running install.py for extension extensions\sd_dreambooth_extension.
Command: "H:\stable-diffusion-webui\venv\Scripts\python.exe" "extensions\sd_dreambooth_extension\install.py"
Error code: 1
stdout: Installing requirements for Dreambooth
Checking Dreambooth requirements.
Checking/upgrading existing torch/torchvision installation
Checking torch and torchvision versions
